benchmark_test/scripts/benchmark-test-setup.sh

```diff
@@ -3,6 +3,11 @@
 set -x
 set -e
 
+# create and clean tmpdir
```

Comment:
Can you explain what this is for?

Reply:
It's just a temp folder for git clone longhorn-manager

Reply:
forgot to add this in the initial commit, so currently longhorn-manager would be cloned into root directory, which is not ideal.

Reply:
What is the purpose of removing the directory immediately after `mkdir`?

Reply:
No need to do this. I've removed this line. Thank you!